test/security/ClientSSLSecurityPFX.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+var fs = require('fs'),
+    join = require('path').join;
+
+describe('ClientSSLSecurityPFX', function() {
+  var ClientSSLSecurityPFX = require('../../').ClientSSLSecurityPFX;
+  var pfx = __filename;
+
+  it('is a function', function() {
+    ClientSSLSecurityPFX.should.be.type('function');
+  });
+
+  describe('defaultOption param', function() {
+    it('is accepted as the second param', function() {
+      new ClientSSLSecurityPFX(null, {});
+    });
+
+    it('is used in addOptions', function() {
+      var options = {};
+      var defaultOptions = { foo: 5 };
+      var instance = new ClientSSLSecurityPFX(null, defaultOptions);
+      instance.addOptions(options);
+      options.should.have.property("foo", 5);
+    });
+  });
+
+  it('should throw if invalid pfk file is given', function () {
+    var instanceCert = null;
+
+    try {
+      instanceCert = new ClientSSLSecurityPFX({});
+    } catch (e) {
+      //should happen!
+      instanceCert = false;
+    }
+
+    if (instanceCert !== false) {
+      throw new Error('accepted wrong pfk');
+    }
+  });
+
+  it('should be able to make a request', function (done) {
```

Comment:
Will `ClientSSLSecurityPFX` make a request, or will the library as a whole?  A more appropriate spec name may be `it('should be usable in a request...`.